src/behaviors/editor-connector.js

```diff
@@ -43,75 +43,74 @@ export default {
         this.importHref(editorUrl, resolve, reject);
       })
       .then(() => {
-        const { Editor, formatters } = window.SimplaText,
-              toFormatter = (command) => formatters[command];
+        const { RichText } = window.SimplaBehaviors;
 
-        return new Editor({
-          dom: this,
+        let editor,
+            updateTools,
+            updateRangeAndTarget;
+
+        editor = new RichText(this, {
           inline: this.inline,
-          formatters: this.commands.map(toFormatter),
           placeholder: this.placeholder,
-          formatterChangedCallback: ({ name }, { applied, meta }) => {
-            this
-              .loadEditor()
-              .then(editor => {
-                if (toolbar.target === editor) {
-                  this._tools = this._tools || {};
-                  this._tools[name] = { applied, meta };
-
-                  if (toolbar.set) {
-                    toolbar.set(`tools.${name}.active`, applied);
-                    toolbar.set(`tools.${name}.meta`, meta);
-                  } else {
-                    toolbar.tools = toolbar.tools || {};
-                    toolbar.tools[name] = { applied, meta };
-                  }
-                }
-              });
-          },
-          selectCallback: (selection) => {
-            const shouldShowToolbar = selection && this.editable && !this.plaintext;
-
-            this.fire('select', { selection });
-
-            if (toolbar.parentElement !== document.body) {
-              document.body.appendChild(toolbar);
+          plugins: this.plugins,
+          editable: this.editable,
+          typeographer: true
+        });
+
+        updateTools = ({ name, applied, meta }) => {
+          if (toolbar.target === editor) {
+            this._tools = this._tools || {};
```

Comment:
What's the point of a separate `_tools` prop, rather than just using `_editor.plugins`?